pkgs/by-name/kr/kryptor/package.nix

```
@@ -11,25 +13,35 @@ buildDotnetModule rec {
   src = fetchFromGitHub {
     owner = "samuel-lucas6";
     repo = "Kryptor";
-    rev = "v${version}";
+    tag = "v${version}";
     hash = "sha256-BxUmDzmfvRelQDHb5uLcQ2YPL7ClxZNFGm/gQoDK8t8=";
   };
 
-  dotnet-sdk = dotnetCorePackages.sdk_6_0;
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
   projectFile = "src/Kryptor.sln";
   nugetDeps = ./deps.nix;
 
-  executables = ["kryptor"];
+  executables = [ "kryptor" ];
 
-  dotnetFlags = ["-p:IncludeNativeLibrariesForSelfExtract=true"];
+  dotnetFlags = [ "-p:TargetFramework=net8.0" ];
+
+  doInstallCheck = true;
+  nativeInstallCheckInputs = [ versionCheckHook ];
+
+  passthru = {
+    updateScript = ./update.sh;
+  };
 
   meta = {
     changelog = "https://github.com/samuel-lucas6/Kryptor/releases/tag/v${version}";
```

@getchoo I also wanted to try this as you suggested in https://github.com/NixOS/nixpkgs/pull/362757#discussion_r1874576247, but `src` doesn't have a `tag` attribute and will cause an eval failure. I assume this is an oversight in the recent implementation
```suggestion
    changelog = "https://github.com/samuel-lucas6/Kryptor/releases/tag/${src.tag}";
```

---

Yup, guess so :(